src/bestpractices/http-caching.md

```diff
@@ -0,0 +1,19 @@
+# HTTP Caching
+
+There are several different "levels" at which you could configure HTTP caching for your site on Platform.sh.  Which one you want to use depends on your specific use case.  You should use only one of these at a time and disable any others.  Mixing them together will most likely result in stale and unclearable caches.
+
+* The Platform.sh [Router cache](/configuration/routes.cache.md).  Every project includes a router instance, which includes optional HTTP caching.  It is reasonably configurable and obeys HTTP cache directives, but does not support push-based clearing.  If you are uncertain what caching tool to use, start with this one.  It is more than sufficient for the majority of use cases.
+
+* A Content Delivery Network (CDN).  Platform.sh is compatible with most commercial CDNs.  If your Platform.sh Enterprise project has a Dedicated production environment it will typically come with the [Fastly CDN](https://docs.platform.sh/golive/steps/fastly.html).  A CDN will generally offer the best performance as it is the only option that includes multiple geographic locations, but it also tends to be the most expensive.  Functionality will vary widely depending on the CDN.  Setup instructions for Fastly and [Cloudflare](https://docs.platform.sh/golive/steps/cloudflare.html) are available, and will be similar for most other CDNs.
```

Comment:
```suggestion
* **A Content Delivery Network (CDN)**.  Platform.sh is compatible with most commercial CDNs.  If your Platform.sh Enterprise project has a Dedicated production environment it will typically come with the [Fastly CDN](https://docs.platform.sh/golive/steps/fastly.html).  A CDN will generally offer the best performance as it is the only option that includes multiple geographic locations, but it also tends to be the most expensive.  Functionality will vary widely depending on the CDN.  Setup instructions for Fastly and [Cloudflare](https://docs.platform.sh/golive/steps/cloudflare.html) are available, and will be similar for most other CDNs.
```